homeassistant/components/xiaomi_miio/config_flow.py

```diff
@@ -146,7 +150,8 @@ async def async_step_device(self, user_input=None):
                     if model.startswith(device_model):
                         unique_id = self.mac
                         await self.async_set_unique_id(unique_id)
```

Comment:
We want to set `raise_on_progress` to `False` here to avoid aborting the user flow if there's a discovery flow for the same device already in progress.

Reply:
@starkillerOG, You might want to re-open this PR and adjust this... As @MartinHjelmare is right.